### PR TITLE
PA Migration: Update docs

### PR DESCRIPTION
**Pull request by fpetrini15**
Point PA docs to new repo

Server changes: https://github.com/triton-inference-server/server/pull/7488
Client changes: https://github.com/triton-inference-server/client/pull/771
PA changes: https://github.com/triton-inference-server/perf_analyzer/pull/21
MA changes: https://github.com/triton-inference-server/model_analyzer/pull/925
PyTriton changes: https://github.com/triton-inference-server/pytriton/pull/80
Tutorial changes: https://github.com/triton-inference-server/tutorials/pull/105

### CODE REVIEW

**Review by szkarpinski — CHANGES_REQUESTED** [review]
@fpetrini15 The new link appears to be broken (https://github.com/triton-inference-server/perf_analyzer/blob/main/README.md) and https://github.com/triton-inference-server/perf_analyzer/ repo seems not to be live yet.

Please let us know when it's there and we'll be happy to merge this. Thanks!

File: benchmarks/BM_efficientnet/README.md [hunk 1]
```diff
@@ -91,4 +91,4 @@ size will be dumped into a separate file, with the pattern: `report-<batch_size>
 all the reports will be combined into one output file per benchmark run in the `combined.csv` file.
 
 Since `perf_analyzer` is used for the benchmark, the meaning of all the values in the report
-may be found in the [`perf_analyzer` documentation](https://github.com/triton-inference-server/client/blob/main/src/c++/perf_analyzer/README.md).
\ No newline at end of file
+may be found in the [`perf_analyzer` documentation](https://github.com/triton-inference-server/perf_analyzer/blob/main/README.md).
```
Comment (mzient) [inline benchmarks/BM_efficientnet/README.md:94]: The page does not exist.
https://github.com/triton-inference-server/perf_analyzer/blob/main/README.md

**Comment by fpetrini15** [comment]
Correct, the repo is still private. We are aiming to switch it to public shortly. I'll let you know when that happens!

**Comment by fpetrini15** [comment]
The repo is now public. Please check again.

**Review by szkarpinski — APPROVED** [review]

**Comment by fpetrini15** [comment]
@szkarpinski You'll have to merge as well. I am not authorized.

**Pull request merged**